examples/swift/hello_world/DemoStringAccessor.swift

```diff
@@ -0,0 +1,9 @@
+import Envoy
+import Foundation
+
+/// Example of a simple HTTP filter that adds a response header.
+struct DemoStringAccessor: EnvoyStringAccessor {
```

Comment:
@rebello95 do I need a swift protocol to be able to use`EnvoyStringAccessor` objc protocol on user defined accessors?

Reply:
Sorry can you elaborate? I'm not sure what you're asking

Reply:
The high level question is: I want DemoStringAccessor to implement the EnvoyStringAccessor interface, but I am not sure how to import EnvoyStringAccessor to this file.

Reply:
Ah, you won't be able to do that because we only expose Swift types externally from `Envoy.framework`. You'd need to declare a `public protocol` in Swift instead. However, per my comments below I still think we should be registering lambdas/closures instead of a type with a `getString()` method

Reply:
@rebello95 finally having the space to come back to this. The idea here is that the `EnvoyStringAccessor` protocol/interface will eventually be made into a generic getter/setter of arbitraty types across the bridge layer (https://github.com/envoyproxy/envoy-mobile/issues/1192). So I wanted to have a concrete type rather than free-floating lambdas. 

However, I can see us exposing the engine builder API as registering a lambda, and then the objc code builds an object around it. The only friction is that I would need to redo the kotlin code to maintain cross-platform homogeneity.

Reply:
Yea personally I'd opt towards the lambda option since that's more consistent with the other APIs Envoy Mobile exposes today (request/response/stream callbacks, for example), and it solves the ambiguity concern around something called `getString()`. @buildbreaker do you have thoughts?

Reply:
@rebello95 Yeah, I remember chatting with @junr03 on this. He wanted to use an object/class here because it allows us to extend/add more accessors for data. Would you be open to a rename of said object instead?

One alternative that I just thought about is if we have a lambda like `()->AccessorObject` and have the `AccessorObject` just be the extensible metadata object.

Reply:
Talked with @junr03 about this a bit more offline (thank you both for responding). I'm ok with this general approach of allowing the consumer to register instances of different types that conform to this relatively generic protocol, assuming the intent is that those types are registered within specific contexts.

Maybe the naming of this would help improve a bit?

```swift
protocol EnvoyStringAccessor: NSObject {
    func envoyStringValue() -> String
}
```

Also it sounds like Java is currently exposing this interface up to the platform; we should switch that over to Kotlin